subgraph.localhost.template.yaml

```diff
@@ -90,6 +90,7 @@ dataSources:
                 handler: handleNewChain
               - event: "Rewarded(indexed uint256,indexed address,indexed address,uint256)"
                 handler: handleRewarded
+                receipt: true
```

Comment:
This is how we ask the Graph Node to fetch the receipt and pass it down to the Subgraph event